### PR TITLE
[NEW UI] Add error layouts for 404 and 500 pages

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add error layouts for 404 and 500 pages in Storybook
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | Check the error layouts in Storybook.

### CODE REVIEW

File: storybook/stories/layouts/Error.stories.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import ErrorLayout from "../../../views/templates/layouts/error.html.twig";
+
+export default {
+  component: ErrorLayout,
+  title: "Layouts/Errors",
+  args: {
+    psBaseUri: "/",
+    error_code: "404",
+  }
+};
```
Comment (Quetzacoalt91) [inline storybook/stories/layouts/Error.stories.js]: Why having this story if the 404 exists as well?

File: views/templates/errors/404.html.twig [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <h2 class="error-page__title">
+    {{ 'Something went wrong...'|trans({}) }}
+  </h2>
+{% endblock %}
+
+{% block description %}
+  <p class="error-page__desc">
+    {{ 'We can’t find your update file in the server.'|trans({}) }}
+    <br/>
+    {{ 'Add a file to restore or update your store.'|trans({}) }}
```
Comment (Quetzacoalt91) [inline views/templates/errors/404.html.twig]: This is usually caused by something else. I would rather say something like "The address you requested cannot be found... bla bla"

Comment (tblivet) [reply views/templates/errors/404.html.twig]: I have updated the wording with the text you proposed on Figma, but I think we may need to make some changes later.

File: views/templates/layouts/error.html.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      {% endblock %}
+
+      {% block button %}
+        <a class="error-page__button btn btn-primary btn-lg" href="#">
+          {{ 'Go back to Update assistant'|trans({}) }}
+        </a>
+      {% endblock %}
+    </div>
+  </div>
+</div>
```
Comment (Quetzacoalt91) [inline views/templates/layouts/error.html.twig:21]: I see this file is like the error 404 template.
Is it necessary to have this duplication? What do you think about having only the skeletons (only the blocks definitions) in this file?

Comment (tblivet) [reply views/templates/layouts/error.html.twig:21]: I have made some changes to harmonize 'skeleton' across Twig files.

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**